kernel/classes/datatypes/ezuser/ezusertype.php

```diff
@@ -258,10 +258,22 @@ function storeObjectAttribute( $contentObjectAttribute )
             }
 
             // saving information in the object attribute data_text field to simulate a draft
-            $contentObjectAttribute->setAttribute( 'data_text', $this->serializeDraft( $user ) );
+            // only if the object version is a draft (status == 0)
+            if(
+                $user->Login &&
+                $contentObjectAttribute->attribute( 'object_version' )->attribute( 'status' ) == 0
```

Comment:
Hope you don't mind if I ignore it. Maybe it's a string and then it would hurt to be extra strict here.

Reply:
We ought to be strict, strictly speaking ;) eZContentObjectVersion says it's an integer. But I see many cases where we aren't strict, so I guess this can pass if others agree.